hie.yaml

```diff
@@ -1 +1,6 @@
-cradle: {stack: {component: "ghcide:lib"}}
```

Comment:
Not sure you meant to commit this.

Reply:
Whoops, I did not. But it is useful for working on the tests. Would anyone object to a separate PR for this?